doc/user/content/sql/appendix-cluster-sizes.md

```diff
@@ -7,8 +7,40 @@ menu:
     weight: 900
 ---
 
+## Default Cluster Sizes
+
 {{% self-managed/materialize-cluster-sizes %}}
 
+## Custom Cluster Sizes
+
+When installing the Materialize Helm chart, you can override the [default
+cluster sizes and resource allocations](/sql/appendix-cluster-sizes/). These
+cluster sizes are used for both internal clusters, such as the `system_cluster`,
+as well as user clusters.
+
+{{< tip >}}
+
+In general, you should not have to override the defaults. At minimum, we
+recommend that you keep the 25-200cc cluster sizes.
+
+{{</ tip >}}
+
+```yaml
+operator:
+  clusters:
+    sizes:
+      <size>:
+        workers: <int>
+        scale: <int>
+        cpu_exclusive: <bool>
+        cpu_limit: <float>         # e.g., 6
+        credits_per_hour: <string> # e.g., "0.0"
```

Comment:
credits_per_hour could be optional? That's a different change, but we should document that it's just a number for accounting purposes.

Reply:
yeah, probably worth back-porting a "0.00" default. I can do this.